audbcards/core/dataset.py

```diff
@@ -1,6 +1,10 @@
+from datetime import datetime
```

Comment:
This is no longer needed, see https://github.com/audeering/audbcards/actions/runs/7989060385/job/21814910422?pr=36

```suggestion
```